frontend/projects/upgrade/src/app/core/http-interceptors/http-error.interceptor.ts

```diff
@@ -17,25 +17,16 @@ export class HttpErrorInterceptor implements HttpInterceptor {
   openPopup(error) {
     const temp = {
       type: NotificationType.Error,
-      title: 'Network call failed.',
+      title: 'Network call failed. See console for details.',
       content: error.url,
       animate: 'fromRight',
     };
-    if (!this.environment.production) {
-      temp.title += ' See console for details.';
-    }
-    if (!(error.status === 401) && !this.environment.production) {
-      this._notifications.create(temp.title, temp.content, temp.type, temp);
-    }
+    this._notifications.create(temp.title, temp.content, temp.type, temp);
   }
 
   intercept(request: HttpRequest<any>, next: HttpHandler): Observable<HttpEvent<any>> {
     return next.handle(request).pipe(
       catchError((err) => {
-        if (err.status === 401) {
-          // auto logout if 401 response returned from api
-          this.authService.authLogout();
```

Comment:
Is this the code that logs out when the server sends a 401 error?

Reply:
yes